packages/functional-tests/tests/oauth/oauthPermissions.spec.ts

```diff
@@ -2,53 +2,43 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { test, expect } from '../../lib/fixtures/standard';
-
-let email;
-const password = 'passwordzxcv';
+import { test, expect, password } from '../../lib/fixtures/standard';
 
 test.describe('severity-1 #smoke', () => {
   test.describe('oauth permissions for trusted reliers - sign up', () => {
+    test.use({
+      emailOptions: [{ prefix: '', password: 'passwordzxcv' }],
```

Comment:
This should use the imported `password` above from L5.
```suggestion
      emailOptions: [{ prefix: '', password }],
```

We might also be able to find a solution where `prefix` is just defaulted to `''` so we don't need to explicitly pass an empty string.

Reply:
Re: default prefix, I made a comment above at the place where I imagined one could possibly default to an empty string. If that actually works, this could just be simplified to this [since the prefix would be optional, like the `newPassword`]:

```js
      emailOptions: [{ password }],
```